PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -195,107 +412,287 @@ struct HfTaskFlow {
+    mixedTPCTPCChChMC.setObject(new CorrelationContainer("mixedTPCTPCChChMC", "mixedTPCTPCChChMC", corrAxis, effAxis, {}));
+  } // End of init() function
 
-  //  ---------------
+  // =========================
   //    templates
   //  FIXME: Some collisions are rejected here, what causes (part of) differences with the D0 task
-  //  ---------------
+  // =========================
   template <typename TCollision>
   bool isCollisionSelected(TCollision const& collision, bool fillHistograms = false)
   {
-    if (processRun2 == true) {
-      //  Run 2: trigger selection for data case
-      if (fillHistograms)
-        registry.fill(HIST("Data/hEventCounter"), 1);
-      if (!processMc) {
-        if (!collision.alias_bit(kINT7)) {
-          return false;
-        }
-      }
-      //  Run 2: further offline selection
-      if (fillHistograms)
-        registry.fill(HIST("Data/hEventCounter"), 2);
-      if (!collision.sel7()) {
-        return false;
-      }
-      if (fillHistograms)
-        registry.fill(HIST("Data/hEventCounter"), 3);
-    } else {
-      //  Run 3: selection
-      if (fillHistograms)
-        registry.fill(HIST("Data/hEventCounter"), 1);
+    if (fillHistograms)
+      registry.fill(HIST("Data/hEventCounter"), 1);
+
+    if (processMc == false) {
       if (!collision.sel8()) {
         return false;
       }
-      if (fillHistograms)
-        registry.fill(HIST("Data/hEventCounter"), 3);
     }
+
+    if (fillHistograms)
+      registry.fill(HIST("Data/hEventCounter"), 3);
+
     return true;
   }
 
+  // =========================
+  //      Quality Assesment plots
+  // =========================
+
+  // ---- DATA : TPC-TPC h-h Same Event QA ----
   template <typename TTracks>
-  void fillQA(float multiplicity, TTracks const& tracks)
+  void fillTpcTpcChChSameEventQA(float multiplicity, TTracks const& tracks)
   {
-    int Ntracks = 0;
+    int nTracks = tracks.size();
     for (const auto& track1 : tracks) {
-      Ntracks++;
       registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hPt"), track1.pt());
       registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hEta"), track1.eta());
       registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hPhi"), track1.phi());
       registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hYields"), multiplicity, track1.pt(), track1.eta());
       registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hEtaPhi"), multiplicity, track1.eta(), track1.phi());
     }
-    registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hNtracks"), Ntracks);
+    registry.fill(HIST("Data/TpcTpc/HadronHadron/SameEvent/hNtracks"), nTracks);
   }
 
+  // ---- MC : TPC-TPC h-h Same Event QA ----
+  // Changed quickly void for int type + added the return for a test
+  template <CorrelationContainer::CFStep step, typename TTracks>
+  int fillTpcTpcChChSameEventQAmc(float multiplicity, TTracks const& tracks)
```

Comment:
```suggestion
  int fillTpcTpcChChSameEventQaMc(float multiplicity, TTracks const& tracks)
```

Reply:
Other QA functions are similarly called "fill.....SameEventQA", should I change to "fill....SameEventQa" for all of them or that recommended change is only for MC functions ?

Reply:
If you can change `QA` -> `Qa` in the other names as well, that would be great, thanks.